### PR TITLE
Ex 05 feedback

### PR DESCRIPTION
**Pull request by jysandy**

### CODE REVIEW

File: src/clojure_by_example/ex05_immutability_and_fp.clj [hunk 1]
```diff
@@ -59,26 +56,25 @@ pi ; evaluate to confirm
 
 ;; - So suppose we define...
 
-(def habitable-planets [{:pname "Earth" :moons 1}
```
Comment (jysandy) [inline src/clojure_by_example/ex05_immutability_and_fp.clj:62]: Calling them `habitable-planets` to begin with adds a lot of confusion at line 81, where you evaluate `habitable-planets` and it returns the planets unchanged, which just looks like all planets are habitable.

File: src/clojure_by_example/ex05_immutability_and_fp.clj [hunk 2]
```diff
@@ -118,7 +114,7 @@ habitable-planets ; confirm by checking the value of this
 ;; At each re-definition, `am-i-mutable` effectively "becomes"
 ;; the new _immutable_ value.
 ;;
-;; "Immutable" value is actually an oxymoron. Recall, we said that
```
Comment (jysandy) [inline src/clojure_by_example/ex05_immutability_and_fp.clj:121]: `"Immutable" value` is a tautology. `"Mutable" value` is an oxymoron. We can choose to use one or the other.

File: src/clojure_by_example/ex05_immutability_and_fp.clj [hunk 3]
```diff
@@ -421,26 +417,31 @@ habitable-planets ; confirm by checking the value of this
 
 ;; We can also alias the whole hash-map:
 
-(let [add-message (fn [{:keys [pname moons]
```
Comment (jysandy) [inline src/clojure_by_example/ex05_immutability_and_fp.clj:424]: These function bodies are starting to get bigger, and putting them in `let` blocks makes them harder to visually parse.

File: src/clojure_by_example/ex05_immutability_and_fp.clj (hunk 2 above)
Comment (adityaathalye) [reply src/clojure_by_example/ex05_immutability_and_fp.clj:121]: Ayyo! Yes, /s/Immutable/Mutable

**Review by adityaathalye — APPROVED** [review]

File: src/clojure_by_example/ex05_immutability_and_fp.clj (hunk 3 above)
Comment (adityaathalye) [reply src/clojure_by_example/ex05_immutability_and_fp.clj:424]: Right, let's pull them out. I can do it... Should I?

File: src/clojure_by_example/ex05_immutability_and_fp.clj (hunk 1 above)
Comment (adityaathalye) [reply src/clojure_by_example/ex05_immutability_and_fp.clj:62]: Agreed.

File: src/clojure_by_example/ex05_immutability_and_fp.clj (hunk 3 above)
Comment (adityaathalye) [reply src/clojure_by_example/ex05_immutability_and_fp.clj:424]: Oh you did it... I missed that. :100:

**Pull request merged**